### PR TITLE
update operator-sdk to use local/bin

### PR DESCRIPTION
**Pull request by yazug**
specify version for operator-sdk for make bundle
download operator-sdk pinned version when generating bundle update make bundle to use local operator-sdk

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -324,6 +348,11 @@ golint: get-ci-tools
 	GOWORK=off PATH=$(GOBIN):$(PATH); $(CI_TOOLS_REPO_DIR)/test-runner/golint.sh
 	GOWORK=off PATH=$(GOBIN):$(PATH); $(CI_TOOLS_REPO_DIR)/test-runner/golint.sh ./api
 
+.PHONY: operator-sdk
+operator-sdk: $(OPERATOR_SDK) ## Download operator-sdk locally if necessary.
+$(OPERATOR_SDK): $(LOCALBIN)
+	test -s $(OPERATOR_SDK) || curl -o $(LOCALBIN)/operator-sdk -L https://github.com/operator-framework/operator-sdk/releases/download/$(OPERATOR_SDK_VERSION)/operator-sdk_linux_amd64 && chmod +x $(LOCALBIN)/operator-sdk
```
Comment (abays) [inline Makefile:354]: I think this is missing some `( )`.  Without them, if `test -s` passes, then it tries to `chmod +x $(LOCALBIN)/operator-sdk`, which may not exist.  This is the error I see:

```
chmod: cannot access '/home/ocp/podified/yazug-openstack-baremetal-operator/bin/operator-sdk': No such file or directory
make: *** [Makefile:354: /home/ocp/bin/operator-sdk] Error 1
```

Comment (abays) [reply Makefile:354]: ...so think we need to put `( )` around:

```
curl -o $(LOCALBIN)/operator-sdk -L https://github.com/operator-framework/operator-sdk/releases/download/$(OPERATOR_SDK_VERSION)/operator-sdk_linux_amd64 && chmod +x $(LOCALBIN)/operator-sdk
```

Comment (yazug) [reply Makefile]: yea fixed, that was a duplicate

**Review by abays — APPROVED** [review]
/lgtm

**Comment by abays** [comment]
recheck

**Pull request merged**